frontend/src/metabase/query_builder/containers/QueryBuilder.unit.spec.tsx

```diff
@@ -283,4 +291,32 @@ describe("QueryBuilder", () => {
       expect(mockEvent.returnValue).not.toEqual(BEFORE_UNLOAD_UNSAVED_MESSAGE);
     });
   });
+
+  describe("renders the row count regardless of visualization type", () => {
+    const dataset = TEST_MODEL_DATASET;
+    const cards = [
+      createMockCard({ ...TEST_CARD_VISUALIZATION, display: "table" }),
```

Comment:
nit: I'd probably add constants like `TEST_CARD_LINE` or whatever and not create them here, but no strong opinion about this